src/components/cart/CartWrapper.tsx

```
@@ -0,0 +1,30 @@
+"use client";
+
+import { useMemo } from "react";
+import CartIcon from "./CartIcon";
+import CartSheet from "./CartSheet";
+import type { ProductCardProps } from "@/components/product/ProductCard";
+import { useAppDispatch, useAppSelector } from "@/hooks/redux-hooks";
+import { closeCart, openCart, updateQuantity, removeItem } from "@/store/cart-slice";
+
+export default function CartWrapper({}: { initialItems?: ProductCardProps[] }) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)